### PR TITLE
make weberror pip installable

### PR DESCRIPTION
**Pull request by aleksandra-tarkowska**
# What this PR does

pip install weberror

# Testing this PR

```
$ mkdir test
$ cd test
$ virtualenv venvweb
$ source venvweb/bin/activate
(venvweb) $ pip install omego 
(venvweb) $ omego download --ice 3.6 --branch OMERO-DEV-merge-build py
(venvweb) $ mv OMERO.py[currentversion] OMERO.py
(venvweb) $ pip install -r OMERO.py/share/web/requirements-py27-trial.txt

(venvweb) $ pip install git+git://github.com/aleksandra-tarkowska/weberror.git@pip_installer
(venvweb) $ OMERO.py/bin/omero config append omero.web.apps '"weberror"'
(venvweb) $ OMERO.py/bin/omero web start
```

then generate nginx config and reload. Web should operate as normal

Full doc in http://www.openmicroscopy.org/site/support/omero5.3-staging/sysadmins/unix/install-web/install-web-trial.html

see https://github.com/openmicroscopy/openmicroscopy/pull/4808 for reference

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+# Author: Aleksandra Tarkowska <A(dot)Tarkowska(at)dundee(dot)ac(dot)uk>,
+#
+# Version: 1.0
+
+
+from setuptools import setup, find_packages
+
+VERSION = '0.0.1'
```
Comment (sbesson) [inline setup.py]: not a blocker but i would start at 0.1.0 as per semver recommendations

**Comment by sbesson** [comment]
As for the other pip-installable repos, I would add `python setup.py sdist install` to Travis.

**Comment by sbesson** [comment]
Also as we have been through this for all Python packaged repos, Pypi does not support markdown in the top-level readme, so we might be up for a round of md -> rst conversion across the board.

**Comment by jburel** [comment]
I will look at the installation step on omero-install

File: README.rst [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+Installation
+============
+
+Install OMERO.web
+
+This app installs into the OMERO.web framework.
+
+::
+
+    $ pip install omeroweb-weberror
```
Comment (sbesson) [inline README.rst]: `omero-weberrror` ?

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+setup(name="omero-weberror",
+      packages=find_packages(exclude=['ez_setup']),
+      version=VERSION,
+      description="A Python plugin for OMERO.web",
+      long_description=read('README.rst'),
+      author='The Open Microscopy Team',
+      author_email='ome-devel@lists.openmicroscopy.org.uk',
+      license='AGPLv3',
+      url="https://github.com/openmicroscopy/weberror",
+      download_url='https://github.com/openmicroscopy/weberror/tarball/0.1.0',  # NOQA
```
Comment (sbesson) [inline setup.py]: Use `%s' % VERSION`

File: README.rst (hunk 2 above)
Comment (aleksandra-tarkowska) [inline README.rst]: sorry ?

Comment (sbesson) [inline README.rst]: The command does not match the Pypi package naming

**Comment by aleksandra-tarkowska** [comment]
Done

**Comment by joshmoore** [comment]
Worth deploying as "omero-weberror" or do we use the git URL for installing as an internal testing app?

**Comment by joshmoore** [comment]
(To be clear, I'd point at a released tag if using an URL)

**Comment by aleksandra-tarkowska** [comment]
it deploys from snoopy merge now

**Comment by joshmoore** [comment]
Testing on PyPi now.

**Pull request merged**